pkg/cmd/context/generate/build-configs.go

```diff
@@ -84,7 +87,7 @@ func BuildConfiguration(svcConfig *servicecontext.ServiceConfig, opts *options)
 		return opts.localizer.MustLocalizeError("context.generate.log.info.noSevices")
 	}
 
-	serviceAccount, err := createServiceAccount(opts, "to-do-generate-description")
+	serviceAccount, err := createServiceAccount(opts, fmt.Sprintf("%s-%v", opts.name, time.Now().Unix()))
```

Comment:
I done some testing I think that might require us to validate name to not have spaces

Reply:
The regexp for context name has been matched with that of service account short description.